actions/actions.py

```diff
@@ -691,9 +691,18 @@ def run(self, dispatcher, tracker, domain):
 
         if alg_res and alg_res.get("hits") and len(alg_res.get("hits")) > 0:
             docs_found = True
-            doc_list = algolia.get_algolia_link(alg_res.get("hits"), 0)
+            hits = [
+                hit
+                for hit in alg_res.get("hits")
+                if "Rasa X Changelog " not in hit.get("hierarchy", {}).values()
+                and "Rasa Open Source Change Log "
+                not in hit.get("hierarchy", {}).values()
```

Comment:
Would it be easier to check the link url for the word `changelog`?

Reply:
I think that would have to be after we `get_algolia_link` i guess

Reply:
yeah, when I was testing I just checked if any hierarchy level had "change log" or "changelog" in it, but then I thought maybe that could end up funky (and for Rasa X, the compatibility matrix falls under a "Changelog" heading, and that might be a legitimate result sometimes)

Reply:
good point!